src/rma/js/components/review-list/review-item-form.js

```diff
@@ -0,0 +1,78 @@
+import React, { useState, useContext } from "react";
+
+import { HiddenInput } from "../../forms/inputs";
+import { ConstantsContext, SuggestionContext } from "./context";
+import { ZaakDetailModal } from "./zaak-detail-modal";
+
+
+const ReviewItemForm = ({ index, data }) => {
+    const { list_item_id, zaak }  = data;
+    const { formsetConfig, zaakDetailPermission } = useContext(ConstantsContext);
+    const { suggestions } = useContext(SuggestionContext);
+
+    const prefix = formsetConfig.prefix;
+    const id_prefix = (field) => `id_${prefix}-${index}-${field}`;
+    const name_prefix = (field) => `${prefix}-${index}-${field}`;
+
+    const [ comment, setComment ] = useState("");
+    const suggestion = suggestions[index];
+    const disabled = !!suggestion;
+
+    const actionIcon = (suggestion === "remove" ? "clear": (suggestion === "change_and_remove" ? "create" : "done"));
+
+    // modal
+    const canOpen = zaakDetailPermission === "True";
```

Comment:
Booleans exist in Javascript :-) why is this not an actual boolean?

Reply:
The value is received from data-* attribute. I thought that they can be only strings. 
https://github.com/GemeenteUtrecht/record-management-app/blob/9782da9da31745dfcd184304df3c3db020e6bbb9/src/rma/js/components/review-list/index.js#L14

Reply:
yeah, you'd have to cast that explicitly... I'll leave it for now.